### PR TITLE
Documenting ucrSaveGraph

### PR DESCRIPTION
**Pull request by N-thony**
@lloyddewit I wrote the documentation for ucrSaveGraph. Please could you peer review this PR? Thanks.

### CODE REVIEW

File: instat/ucrSaveGraph.vb [hunk 1]
```diff
@@ -14,73 +14,101 @@
...
+    Private Sub ucrInputGraphName_NameChanged() Handles ucrInputGraphName.NameChanged
+        If chkSaveGraph.Checked Then
+            RaiseEvent GraphNameChanged()
+        End If
+    End Sub
+    ''' <summary> Handles event triggered when the combo box contents changes. </summary>
     Private Sub ucrInputGraphName_ContentsChanged() Handles ucrInputGraphName.ContentsChanged
         RaiseEvent ContentsChanged()
     End Sub
+    ''' <summary> Save the graph when the check box save graph is checked. </summary>
```
Comment (lloyddewit) [inline instat/ucrSaveGraph.vb:92]:
```suggestion
   ''' <summary> Returns the state of the save graph check box. </summary>
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@N-thony This looks great, I just had a couple of small suggestions

**Comment by N-thony** [comment]
@lloyddewit  I solved the issues you raised on 18 October. Please could you confirm that your comments are resolved? thanks.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern please could you approve (functionality not affected, just changes to code comments)? thanks

**Review by rdstern — APPROVED** [review]

**Pull request merged**